src/app/plugins/wolframalpha.plugin.ts

```diff
@@ -0,0 +1,92 @@
+import { MessageAttachment, MessageEmbed } from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IMessage } from '../../common/types';
+import Environment from '../../environment';
+const WolframAlphaAPI = require('wolfram-alpha-api');
+
+type EmbedTypes = 'short' | 'image' | 'error';
+
+export class WolframAlphaPlugin extends Plugin {
+  private _defaultQuestion = 'What can you do?';
+  private _imageOption = ['image', 'img'];
+  private _readableImageOption = this._imageOption.join(' | ');
+  private _logoURL = 'https://www.symbols.com/images/symbol/2886_wolfram-alpha-logo.png';
+  private _errorMessage = "Sorry, I don't know that one";
```

Comment:
Put these under the public constants

Reply:
`usage` depends on some of these fields, so they have to be declared before.